src/NzbDrone.Core/DecisionEngine/Specifications/RssSync/HistorySpecification.cs

```diff
@@ -60,9 +60,9 @@ public virtual Decision IsSatisfiedBy(RemoteBook subject, SearchCriteriaBase sea
                         continue;
                     }
 
-                    // The author will be the same as the one in history since it's the same book.
-                    // Instead of fetching the author from the DB reuse the known author.
-                    var preferredWordScore = _preferredWordServiceCalculator.Calculate(subject.Author, mostRecent.SourceTitle);
+                    // The artist will be the same as the one in history since it's the same album.
+                    // Instead of fetching the artist from the DB reuse the known artist.
```

Comment:
artist and album references, change these back

Reply:
incoming commit

Reply:
Commited